mappings/net/minecraft/util/dynamic/DynamicDeserializer.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3817 net/minecraft/util/DynamicDeserializer
+CLASS net/minecraft/class_3817 net/minecraft/util/dynamic/DynamicDeserializer
```

Comment:
if you create a dynamic package, it should rather go under `net/minecraft/datafixer/dynamic`

Reply:
`Dynamic`s, even though a part of DFU, are used in many places in the codebase.